cellSAM/model.py

```diff
@@ -49,30 +49,72 @@ def get_local_model(model_path: str) -> nn.Module:
     return model
 
 
-def get_model(model: nn.Module = None) -> nn.Module:
+def get_model(model="cellsam_general", version=None) -> nn.Module:
     """
-    Returns a loaded CellSAM model. If model is None, downloads weights and loads the model with a progress bar.
+    Returns a loaded CellSAM model.
+
+    If pretrained model weights specified by ``version`` are not found locally,
+    they will be downloaded from users.deepcell.org.
+
+    Parameters
+    ----------
+    model : str, default="cellsam_general"
+       Which model to load. Options include:
+
+        - ``"cellsam_general"``
+        - ``"cellsam_optimized"``
+
+       ``"cellsam_general"`` is trained only on publicly-available datasets and
+       is made available for reproducibility. Use this model e.g. to reproduce
+       the model evaulation cited in the publication.
+
+       ``"cellsam_optimized"`` incorporates additional non-public training data
+       and therefore is recommended for the standard use-case (i.e. applying
+       to new images).
```

Comment:
@damaggu @ulisrael @rdilip @m274d - can someone double-check my wording here? I'm trying to describe for the user the distinction between the two sets of pretrained weights we're providing, and my own understanding may be incomplete.

Note that `cellsam_general` maps to the `cellsam_general` weights from #84 (with hash beginning `491d2f`) and `cellsam_optimized` (naming arbitrary - feedback welcome) maps to the "neurips" model, hash `931e88f`.